### ISSUE TITLE
[EDIT]  Speeding Up Clips Does not Work (Linux Version)

### ISSUE THREAD

**Issue by Rstation4** [olive-editor/olive#1645]
Linux Mint 20.1  Cinnamon (64 Bit version)
Olive: Olive-45db6f45-Linux-x86_64

Trying to speed up a clip in editing, but the feature does not work in Linux. I see Windows versions showing a ease of use right clip to simply get the job done. I've tried several speeds but nothing seems to work. The timeline doesnt even cache it in the video preview when I tried the speed change at 200% for my clip.

**Comment by itsmattkc** [comment]
Are you certain you're setting the video clip's speed as opposed to the audio clip's?

**Comment by Rstation4** [comment]
When looking at the properties box it is a bit confusing, I ended up just disabling the audio for that segment. Actually features of the video editor in general is confusing, It didn't seem like it was speeding up in the video preview so what I did was also shorten the length of that clip as well to get the desired effect of a clip that was sped up. It does feel like I had to do a more complicated way to simply speed up a clip in the end.
I've seen no proper documentation for it and it refers to Reddit for answers but Reddit in general is just toxic garbage.
